lib/chef/knife/bmcs_common_options.rb

```diff
@@ -9,6 +9,10 @@ class Knife
     module BmcsCommonOptions
       def self.included(includer)
         includer.class_eval do
+          option :region,
+                 long: '--region REGION',
+                 description: 'The region to make calls against.  eg. us-ashburn-1'
```

Comment:
nitpick - to make this consistent with documentation in other BMCS tools: 'The region to make calls against (e.g., `us-ashburn-1`).'